### DISCUSSION TITLE
How can I set div as the center of the screen?

### DISCUSSION THREAD

**Discussion by MinalPatilRPA** [tailwindlabs/tailwindcss#12571] [Help]
I want to div as the center of the screen, as shown in the below image.

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/7def486e-0e7f-4316-9e8a-7fc9eb76510d)

but my output is, as shown in the below 2 image.

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/c35b34db-f07b-4ea2-8114-da16b62921d2)

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/b56430fa-2049-4fe9-aca1-0893bbb89e5a)

my code is: 

  <div className='  grid grid-rows-2 pt-10 items-center'>
          <h1 className='text-white flex flex-row items-center justify-start text-lg pl-[360px] '>Please insert a valid You Tube video URL</h1>
        
          <div className='px-12 flex flex-row justify-center items-center gap-3 '>
              <input placeholder='https://www.youtube.com/watch?v=foo5ZtvVo5I' className='text-white p-3 w-[40%] border border-emerald opacity-30'/>
              <button className='bg-emerald-400 p-3 w-[10%]'>Convert</button>
          </div>
      </div>
      

how can I solve this problem?

**Comment by Zehad-10** [comment]
Hi,

Can you please add your code in https://play.tailwindcss.com/ and share with us that code?

**Comment by MinalPatilRPA** [comment]
Its solve Thanks
